lib/order.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'menu'
+require_relative 'order_item'
+require_relative 'send_text'
+class Order 
+
+attr_reader  :dishes_ordered, :menu, :total, :text_message
+
+    def initialize(menu = Menu.new, total = 0, text_message = TwilioClass.new, order_item = OrderItem)
+        @dishes_ordered = []
+        @total = total
+        @menu = menu
+        @text_message = text_message
+        @order_item = order_item
+    end
+
+    def display_menu
+        menu.display
+    end
+
+    def add(dish_id, quantity)
+        if menu.price_exist(dish_id)
```

Comment:
Easy to understand the logic, maybe in the future when we are more pros, we could refactor and try to make the method be less lengthy